Automattic/wp-calypso
# Jetpack: roll out the treatment variation of the cart dropdown test

#### Changes proposed in this Pull Request

The treatment variation of the `calypso_jetpack_cart_with_dropdown_variant_selector_2 ` experiment was a winner.
This change rolls out that winning variation.

The experiment was initiially implemented in https://github.com/Automattic/wp-calypso/pull/61174.

P2 discussion --> pbxNRc-1rJ-p2#comment-3191

#### Testing instructions

* Boot up Calypso locally, or use the calypso.live link.
* For a Jetpack site that's connected to WordPress.com, add a Jetpack product to the cart.
* On the checkout page you should see a dropdown selector to choose the term (monthly or yearly), and not radio buttons.
* Add a product to the cart for a non-Jetpack (WordPress.com) site and make sure it still displays as it is on production.

<img width="673" alt="Screen Shot 2022-04-19 at 12 25 08 PM" src="https://user-images.githubusercontent.com/690843/164080359-db5a0c64-8ee0-43a1-9dba-d2ba740c9baa.png">
